script/database_role/db_readonly_role.sql

```diff
@@ -0,0 +1,7 @@
+CREATE ROLE {project}_{environment}_readonly;
```

Comment:
great thanks! May I request a 2nd role: `db-app` which grants everything a web-app would need: read/write/delete/... basically any regular operation except for the root-level dangerous stuff such as `drop db`, `truncate`, ...

Reply:
@swiknaba  
How about this?

```
CREATE ROLE {project}_{environment}_app;
ALTER ROLE {project}_{environment}_app WITH LOGIN ENCRYPTED PASSWORD '{generate secure password}';

GRANT CONNECT ON DATABASE {project}_{environment} TO {project}_{environment}_app;
GRANT USAGE ON SCHEMA public TO {project}_{environment}_app;
GRANT ALL ON SCHEMA public TO {project}_{environment}_app;
GRANT ALL ON ALL SEQUENCES IN SCHEMA public TO {project}_{environment}_app;
GRANT INSERT, SELECT, UPDATE, DELETE, REFERENCES, TRIGGER ON ALL TABLES IN SCHEMA public TO {project}_{environment}_app;
```